rikaicontent.js

```diff
@@ -321,19 +321,23 @@ var rcxContent = {
 	keysDown: [],
 	lastPos: { x: null, y: null},
 	lastTarget: null,
+	altGraphKey: false,
 
 	onKeyDown: function(ev) { rcxContent._onKeyDown(ev) },
 	_onKeyDown: function(ev) {
 		//console.log("key=" + ev.key + " keyCode=" + ev.keyCode + ' charCode=' + ev.charCode + ' detail=' + ev.detail);
  
+		if (ev.key === "AltGraph") {
+			this.altGraphKey = true;
+		}
 		if (window.rikaichan.config.showOnKey !== "" && (ev.altKey || ev.ctrlKey)) {
 			if (this.lastTarget !== null) {
 				//console.log(ev);
 				var myEv = {
 					clientX: this.lastPos.x,
 					clientY: this.lastPos.y,
 					target: this.lastTarget,
-					altKey: ev.altKey || ev.key == "Alt",

```

Comment:
If we want altGraph to be the same as alt here why don't we do something like: ev.altKey || ev.modifierAltGraph

That should work if I'm reading the spec correctly.

I'm not sure we need to add a special variable.